recipes/cmake/3.x.x/conanfile.py

```diff
@@ -1,20 +1,24 @@
-import os
 from conan import ConanFile
+from conan.tools.files import chdir, copy, rmdir, get, files
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.build import build_jobs, cross_building, check_min_cppstd
 from conan.tools.scm import Version
-from conan.tools.files import rmdir, get
-from conans import tools, AutoToolsBuildEnvironment, CMake
-from conan.errors import ConanInvalidConfiguration, ConanException
+from conan.errors import ConanInvalidConfiguration
+import os
+
 
-required_conan_version = ">=1.49.0"
+required_conan_version = ">=1.53.0"
```

Comment:
Why does it need 1.53.0? 1.50.0 seems to be enough

Reply:
It's orthogonal. required_conan_version should be the minimum conan version being able to execute this conanfile, not conan version running in c3i.

Reply:
Hi @SpaceIm,
As I understand it from @prince-chrismc, the fact that c3i is not testing with 1.55 is an issue with the rate at which the build team can deploy new updates rather than a desired or intended state. Given the overarching goal of getting to Conan 2.0 ASAP, coupled with the onerous (and questionable) requirement of maintaining backwards compatibility with 1.x, (versus simply branching the recipes as the actual Conan source did), keeping up with the latest 1.x release is critical to not compromising the 2.0 migration work.

Reply:
As I said it's orthogonal, please change required_conan_version to >=1.50.0

Reply:
Hi @prince-chrismc, HI @memsharded,
Can you provide clarity around the CCI required_conan_version strategy? Isn't the goal to move everything to 2.0 and to leverage the new features and thus qualify with the latest versions of Conan 1.x and Conan 2.0? If not, I guess I am confused as to how the CI system is actually validating recipes unless it is looking at the required_conan_version value and using the specified version of Conan for qualification testing, which, based on my understanding of the infrastructure, it is not that sophisticated, but it would be a nice feature.

Reply:
As far as we are concerned from our side, we appreciate that this is a community-lead effort, and we do not expect contributors to spend significant effort in working out the exact minimum Conan version with which a recipe is compatible, although we do please ask that it is not any lower than what is actually needed, so that users will get a clear error that they can take resolve for themselves if they are using a version of Conan that is too old for the recipe. Indeed, our CI does not check this so this is a best-effort situation.

If this recipe  were to be merged with a version higher than strictly necessary I would not expect significant disruption at this point - given that theres a great number of recipes already requiring 1.53, and we do expect users of Conan Center to be on recent versions - chances are most users already are, making this a moot point by now.

On the other hand, if a reviewer kindly spots that this as an improvement, there's no harm in addressing it. 

CC @prince-chrismc

Reply:
Couldn’t agree more, it's best effort to accurately get the true "minimum" and a good default is the one being tested in CCI. Most often we are updating recipes to that latest version so generally those coincide.

I like to drop links with "hey this looks like the most recent feature being used" such that I can politely get them lower it when I know it's too high. If a reviewer shows an accurate version, that would improve the quality of the recipe making it more widely available which is not a bad things.

Reply:
To be transparent, the behind the scenes we've had lots of conversations about the role of the linters. Just my quick summary

- Hooks: validate recipe folder and package contents + those match the recipe's implementation themselves.
- Pylinter: Ensures the quality of recipes by enforcing best practices and proven conventions

I personally would love to see the pylint improved with a table of "these imports were added in this version" and double check that and report an error. However unless it's in the linter and enforced (and my example is not enough to guarantee that) its best effort to get the exactly right one.

we kindly ask contributors do their best and reviewers to helpfully share details they know so together we get the best recipe.